src/features/query-bar/__tests__/FilterButton-test.js

```diff
@@ -1,26 +1,26 @@
 // @flow
 import * as React from 'react';
 
-import {
-    columnForTemplateFieldName,
-    columnForDateType,
-    conditions as fixtureConditions,
-    initialCondition,
-} from '../components/fixtures';
+import { columnForTemplateFieldName, columnForDateType } from '../components/fixtures';
 import FilterButton from '../components/filter/FilterButton';
+import type { ConditionType } from '../flowTypes';
 import { EQUALS, LESS_THAN } from '../constants';
 
+const validCondition: ConditionType = { columnId: '3', id: '0', operator: EQUALS, values: [1] };
```

Comment:
'3' should be replaced with columnForTemplateFieldName.id (or whatever column uses id=3).